packages/wonder-blocks-form/components/radio-group.js

```diff
@@ -56,6 +56,11 @@ type RadioGroupProps = {|
      * Value of the selected radio item.
      */
     selectedValue: string,
+
```

Comment:
I wonder if it would be worth it to have a `TestIdProps` Type that we import and apply, like we do with AriaProps. It would make future changes simpler and put the docs all in one place.